ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -792,6 +856,102 @@ static UnsignedLong get_domain(Fork fork, UnsignedLong epoch, int domain_type) {
         .plus(UnsignedLong.valueOf(domain_type));
   }
 
+  /**
+   * Extract the bit in ``bitfield`` at position ``i``.
+   *
+   * @param bitfield - The Bytes value that describes the bitfield to operate on.
+   * @param bitPosition - The index.
+   */
+  public int get_bitfield_bit(Bytes bitfield, int bitPosition) {
+    return (bitfield.get(bitPosition / 8) >> (7 - (bitPosition % 8))) % 2;
+  }
+
+  /**
+   * Verify ``bitfield`` against the ``committee_size``.
+   *
+   * @param bitfield
+   * @param committee_size
+   */
+  public boolean verify_bitfield(Bytes bitfield, int committee_size) {
+    if (bitfield.size() != (committee_size + 7) / 8) return false;
+
+    for (int i = committee_size; i < bitfield.size() * 8; i++) {
```

Comment:
This doesn't seem to match the spec.  My interpretation of v0.1 of the spec would translate to this:

`for (int i = committee_size + 1 ; i < committee_size - committee_size % 8 +8; i++)`

https://github.com/ethereum/eth2.0-specs/blob/v0.1/specs/core/0_beacon-chain.md#verify_bitfield :

Reply:
Ah - I had this before, changed it because of the changes in master, not v0.1. Should I change it back?

Reply:
For reference: https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#verify_bitfield

Reply:
Gotcha.  Ya it would be better to target a specific release version.  Not that what you did was wrong, but it will help when performing a diff between versions and estimating level of effort.  Later on, it will be handy to have builds that correspond with specific spec versions so that we can reliably interop between clients based on their 'advertised' build